src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -185,8 +185,8 @@ private void createDatePanel() {
 //         JLabel timeLabel = new JLabel(timeStr);
         
         // Example of String concatenation below        
-        StringBuffer timeSB = new StringBuffer("Time:");
-        timeSB.append(I18nUtils.getLocale());
+        StringBuffer timeSB1 = new StringBuffer("Time:");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3AS1149)